tests/Resources/ti.platform.test.js

```diff
@@ -215,7 +215,13 @@ describe('Titanium.Platform', function () {
 
 	it('.id', () => {
 		should(Ti.Platform).have.readOnlyProperty('id').which.is.a.String();
-		// TODO Verify format?!
+		if (OS_IOS) {
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.platform.test.js;
  Line: 218;
-->
- :no_entry_sign: tests/Resources/ti.platform.test.js line 218 – 'OS_IOS' is not defined. (no-undef)